### PR TITLE
Homepage style improvements

### PR DESCRIPTION
**Pull request by xjensen**
Iterative improvements to homepage styles. Getting into the small stuff.

### CODE REVIEW

File: docs/site/_includes/menu.njk [hunk 1]
```diff
@@ -64,7 +64,6 @@
             <div class="expanded-menu-dropdown">
               <a class="expanded-menu-dropdown-link js-event-hm-menu" href="/style/content/" tabindex="-1">Content style guide</a>              
               <a class="expanded-menu-dropdown-link js-event-hm-menu" href="/style/design/" tabindex="-1">Design style guide</a>
-              <a class="expanded-menu-dropdown-link js-event-hm-menu" href="/style/engineering/" tabindex="-1">Engineering style guide</a>
```
Comment (zakiya) [inline docs/site/_includes/menu.njk:67]: Do we need tabindex here?

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Review by chachasikes — APPROVED** [review]
Thank you for rocking all these changes!

**Review by aaronhans — APPROVED** [review]
thanks Jon

File: docs/site/_includes/menu.njk (hunk 1 above)
Comment (xjensen) [reply docs/site/_includes/menu.njk:67]: I think so. The `tabindex=-1` is removed by JS when the dropdown menu is expanded. In this default state, it prevents keyboard navigation from focusing on links that have not yet been revealed.

**Pull request merged**